jib-core/src/main/java/com/google/cloud/tools/jib/registry/ManifestPuller.java

```diff
@@ -71,7 +72,12 @@ public BlobHttpContent getContent() {
     if (manifestTemplateClass.equals(OCIManifestTemplate.class)) {
       return Collections.singletonList(OCIManifestTemplate.MANIFEST_MEDIA_TYPE);
     }
+    if (manifestTemplateClass.equals(V22ManifestListTemplate.class)) {
+      return Collections.singletonList(V22ManifestListTemplate.MANIFEST_MEDIA_TYPE);
+    }
 
+    // why is V22ManifestListTemplate missing in the default behavior? We don't explicitly accept
+    // it, we only handle it if referenced by sha256 (see getManifestTemplateFromJson)
```

Comment:
This comment is confusing.  Is this a dialogue with the reader? Or a todo/wtf?

Reply:
This intention is that we'll only include `V22ManifestListTemplate` in the defaults here when we offer support for building for an alternative platform?

Reply:
Right, I can update the comment to be more clear.